addons/product/test/product_pricelist.yml

```diff
@@ -51,9 +51,9 @@
 -
   !python {model: product.product}: |
     from openerp.tools import float_compare
-    context.update({'quantity':1, 'date': False, 'partner': ref('base.res_partner_4'), 'pricelist': ref("supplier_pricelist")})
+    context.update({'quantity':1, 'date': False, 'partner': ref('base.res_partner_4')})
     product = self.browse(cr, uid, ref("product_product_6"), context=context)
-    assert float_compare(product.price, 792, precision_digits=2) == 0, "Wrong cost price: LCD Monitor."
+    assert float_compare(product.standard_price, 800, precision_digits=2) == 0, "Wrong cost price: LCD Monitor."
```

Comment:
why this change ? product.price is compute field that why we checked here. why it's change to standard_price ?